KNOWN_ISSUES.md

```diff
@@ -369,3 +369,68 @@ E0413 12:28:25.449973 1 authentication.go:65] Unable to authenticate the request
 - See [Why can't I collect metrics from containers, pods, or nodes using Metrics Server in Amazon EKS?] for more details.
 
 [Why can't I collect metrics from containers, pods, or nodes using Metrics Server in Amazon EKS?]: https://aws.amazon.com/premiumsupport/knowledge-center/eks-metrics-server/
+
+## error: Metrics API not available
```

Comment:
Isn't it the same issue as `Network problem when connecting with Kubelet`? one of the causes is the same the incorrect calico IP assignment?

Reply:
@serathius sort of. In this case I don't any get information, just `error: Metrics API not available` and in the calico logs I've seen a bunch of `10.10.157.127:42644->10.96.0.1:443: read: connection reset by peer`.

```
2024-06-19 14:56:01.767 [ERROR][1] main.go 279: Failed to verify datastore error=Get "https://10.96.0.1:443/apis/crd.projectcalico.org/v1/clusterinformations/default": context deadline exceeded
2024-06-19 14:56:09.770 [ERROR][1] main.go 297: Received bad status code from apiserver error=Get "https://10.96.0.1:443/healthz": context deadline exceeded status=0
2024-06-19 15:37:00.121 [ERROR][1] client.go 295: Error getting cluster information config ClusterInformation="default" error=Get "https://10.96.0.1:443/apis/crd.projectcalico.org/v1/clusterinformations/default": context deadline exceeded - error from a previous attempt: read tcp 10.10.157.127:42644->10.96.0.1:443: read: connection reset by peer
2024-06-19 15:37:00.121 [ERROR][1] main.go 279: Failed to verify datastore error=Get "https://10.96.0.1:443/apis/crd.projectcalico.org/v1/clusterinformations/default": context deadline exceeded - error from a previous attempt: read tcp 10.10.157.127:42644->10.96.0.1:443: read: connection reset by peer
```

I believe it's worth the PR.